Update docs about user server culling (max-age now 7 days, inactivity workaround) (#193)
2i2c-org/docs: admin/howto/control-user-server.md

To ensure efficient resource usage, user servers without interactive usage for a
period of time (default `1h`) are automatically stopped (via
[jupyterhub-idle-culler]).
This means your user server might be stopped for inactivity even if you have
a long running process in the notebook. This timeout can be configured.

% TODO: Add link to SRE guide on how to configure this, once it exists

"Culling a user server" has the same effect as [stopping a user
server](user-server/stopping), but should be distinguished from "culling a
kernel" inside a user server. [jupyterhub-idle-culler] is only involved in
culling user servers, not culling kernels.

```{note}

> A crude way to ensure that a server isn't stopped for inactivity is can be read
> about [here](https://github.com/jupyterhub/jupyterhub-idle-culler/issues/55#issuecomment-1413510651).

Reviewer: As an FYI, generic hyperlink text such as "click here" or "read more" is poor for web accessibility and those using assistive technology: https://business.scope.org.uk/article/how-to-write-better-link-text-for-accessibility

```suggestion
A crude way to [ensure that a server isn't stopped for inactivity](https://github.com/jupyterhub/jupyterhub-idle-culler/issues/55#issuecomment-1413510651) can be implemented.
```
